packages/ui-components/src/components/text-field/text-field.scss

```diff
@@ -0,0 +1,115 @@
+@import '../../assets/styles/globals';
+
+@mixin ghost-animation {
```

Comment:
create a `styles/kv-mixins.scss` file and include the file in`styles/globals.scss` because this mixin may be needed in other components